scalyr_agent/agent_main.py

```diff
@@ -368,6 +368,8 @@ def __perform_config_checks(self, no_check_remote):
             finally:
                 client.close()
 
+            print "Server connection verified."
```

Comment:
This will emit the output to stdout even if the user has selected the `quiet` option.  I have a suggest below where we can do what you are trying to do, but in a little more compact of a form.  If you follow that suggestion, you can remove this line.